src/main/java/com/mathworks/ci/BuildArtifactAction.java

```diff
@@ -61,14 +61,19 @@ public String getDisplayName() {
     @CheckForNull
     @Override
     public String getUrlName() {
-        return "buildresults" + this.actionID ;
+        return (this.actionID == null) ? "buildresults" : "buildresults" + this.actionID ;
```

Comment:
```suggestion
        return "buildresults" + (this.actionID == null ? "" : this.actionID);
```

Reply:
why to send buildresults+"" ?

Reply:
The suggested change is to avoid using the string literal `"buildresults"` twice